frontend/src/components/pages/CreateReview/CreateReviewPage.tsx

```diff
@@ -0,0 +1,276 @@
+  const [books, setBooks] = useState<Book[]>([]);
+  const [review, setReview] = useState("");
+  const [featured, setFeatured] = useState("0");
+
+  // const onBookModalClose = () => setShowBookModal(false);
+  const onDeleteBookModalClose = () => setShowDeleteBookModal(false);
+  const onPublishModalClose = () => setShowPublishModal(false);
+  const onDeleteReviewModalClose = () => setShowDeleteReviewModal(false);
+
+  // const handleTagSelected = (e: Option[]) => {
+  //   setTagsSelected(e);
+  // };
+
+  // adds a book to the list of books
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const addBook = (book: Book) => {
+    setBooks([...books, book]);
+  };
+
+  // deletes a book with the given index from the list of books
+  const deleteBook = (index: number) => {
+    setBooks(books.filter((_, i) => i !== index));
+  };
+
+  // updates a book with the given index from the list of books
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const editBook = (book: Book, index: number) => {
+    deleteBook(index);
+    addBook(book);
+  };
+
+  // useEffect hook adds dummy data to the books array
+  useEffect(() => {
+    setBooks([
+      {
+        title: "Shadow Chasers",
+        coverImage:
+          "https://images-na.ssl-images-amazon.com/images/I/51WCelNAo1L.jpg",
+        titlePrefix: "The",
+        seriesOrder: "1",
+        illustrator: ["Spencer Fencer"],
+        translator: ["Translator Alligator"],
+        formats: [
+          {
+            format: "Soft cover",
+            isbn: "42069",
+            price: "13",
+          },
+        ],
+        minAge: 4,
+        maxAge: 8,
+        authors: [
+          {
+            fullName: "Sophia Spencer",
+            displayName: "Sophia Spencer",
+            attribution: "Author",
+          },
+        ],
+        publishers: [
+          {
+            fullName: "Publisher Fisher",
+            publishYear: 2012,
+          },
+        ],
+        seriesName: "The Amazing Series",
+      },
+    ]);
+  }, []);
+
+  return (
+    <Box>
+      {/* <BookModal
+        isOpen={showBookModal}
+        onClose={onBookModalClose}
+        tagsSelected={tagsSelected}
+        handleSelected={handleTagSelected}
+        booksAdded={books}
+        handleBooksAdded={setBooks}
+      /> */}
+      <DeleteModal
+        isOpen={showDeleteBookModal}
+        onClose={onDeleteBookModalClose}
+        bookIndex={deleteBookIndex}
+        deleteBook={deleteBook}
+      />
+      <PublishModal
+        isOpen={showPublishModal}
+        onClose={onPublishModalClose}
+        publishBook={() => {}}
+      />
+      <DeleteReviewModal
+        isOpen={showDeleteReviewModal}
+        onClose={onDeleteReviewModalClose}
+        deleteReview={() => {}}
+      />
+      {/* Tool bar */}
+      <Box
+        display="flex"
+        flexDirection="row"
+        alignItems="center"
+        justifyContent="space-between"
+        m="0"
+        mb="60px"
```

Comment:
Is it possible to combine these two into one prop? I'm not super familiar with Chakra for this case so I could be wrong.